src/components/Footer/FooterLinks/FooterLinksItem.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import cx from 'classnames';
+
+type FooterLinksItemProps = {
+  className?: string;
+  rel?: string;
+  role?: string;
+  tabIndex?: number;
+  target?: string;
+  text: string;
+  url: string;
+};
+
+export const FooterLinksItem = ({
+  className,
```

Comment:
This component is pretty specific - don't think it will need a `className` prop!

Reply:
One of the item has extra class attached to it so we will need this.

Reply:
Is it the _Grant Tracker_ link? If so we should handle these external link classnames differently going forwards anyway, so let's remove

Reply:
I have remove the className prop.